bndtools.core/src/bndtools/editor/completion/BndCompletionProcessor.java

```diff
@@ -45,15 +50,13 @@
     }
 
     private static ICompletionProposal[] proposals(String prefix, int offset) {
-        ArrayList<ICompletionProposal> results = new ArrayList<ICompletionProposal>(Syntax.HELP.size());
-        for (Syntax s : Syntax.HELP.values()) {
-            if (prefix == null || s.getHeader()
-                .startsWith(prefix)) {
-                IContextInformation info = new ContextInformation(s.getHeader(), s.getHeader());
-                String text = prefix == null ? s.getHeader()
-                    : s.getHeader()
-                        .substring(prefix.length());
-                results.add(new CompletionProposal(text + ": ", offset, 0, text.length() + 2, null, s.getHeader(), info, s.getLead())); //$NON-NLS-1$
+        List<ICompletionProposal> results = new ArrayList<>();
+
+        for (String s : ALL_OPTIONS) {
+            if (prefix == null || s.startsWith(prefix)) {
+                IContextInformation info = new ContextInformation(s, s);
+                String text = prefix == null ? s : s.substring(prefix.length());
+                results.add(new CompletionProposal(text + ": ", offset, 0, text.length() + 2, null, s, info, null));
```

Comment:
I'm not sure it's desired to switch the lookup to not use Syntax.HELP.

Why not just add the missing properties to the HELP, along with some helpful information? You can draw that form the existing docs.

Reply:
The following list comprises the instructions that are missing in `Syntax`.

```
CONSUMER_POLICY
NOBUNDLES
PROVIDER_POLICY
RUNSYSTEMCAPABILITIES
REPORTNEWER
UNDERTEST
TESTPACKAGES
NOMANIFEST
SAVEMANIFEST
WAB
WABLIB
RUNFRAMEWORK
RUNFW
RUNKEEP
RUNTRACE
TESTCONTINUOUS
SNAPSHOT
NAMESECTION
DIGESTS
DSANNOTATIONS
DSANNOTATIONS_OPTIONS
BASELINEREPO
EXECUTABLE
RUNNOREFERENCES
JAVAAGENT
STRICT
DIFFIGNORE
DIFFPACKAGES
CONTRACT
NOBUILDINCACHE
EXTENSION
NOJUNIT
NOJUNITOSGI
PREPROCESSMATCHERS
OUTPUTMASK
TESTUNRESOLVED
RUNJDB
RUNENV
EEPROFILE
EXPORT
GESTALT
BNDDRIVER
CHECK
DISTRO
METATYPE_ANNOTATIONS
METATYPE_ANNOTATIONS_OPTIONS
PACKAGEINFOTYPE
JAVAC_SOURCE
JAVAC_TARGET
JAVAC_PROFILE
JAVAC
JAVA
JAVA_DEBUG
EXPORTTYPE
RUNREMOTE
AUGMENT
REQUIRE_BND
GROUPID
IGNORE_STANDALONE
RUNREPOS
INIT
MAVEN_RELEASE
BUILDREPO
CONNECTION_SETTINGS
RUNPROVIDEDCAPABILITIES
WORKINGSET
RUNSTORAGE
INCLUDEPACKAGE
CDIANNOTATIONS
REMOTEWORKSPACE
SIGNATURE_TEST
```